elements/pfe-autocomplete/src/pfe-search-droplist.scss

```
@@ -12,6 +12,10 @@
   display: block;
 }
 
+:host([debug]) [aria-selected] {
+  box-shadow: inset 0 0 0 2px red;
```

```suggestion
  box-shadow: inset 0 0 0 pfe-var(ui--border-width--md) pfe-var(feedback--critical);
```

If you still need this code in the production version, can you update it to use the variables?  If not, feel free to delete.

---

@castastrophe good call. I am going to go ahead and keep the styles since it is really helpful for debugging. I will get the color updated to the appropriate pfe-vars. 🙂